R/import.R

```diff
@@ -17,5 +17,9 @@ mintr_db_download <- function(path, quiet = FALSE) {
 
 
 mintr_data_version <- function() {
-  readLines(mintr_path("version"))
+  readLines(mintr_path("data_version"))
+}
+
+mintr_version <- function() {
+  readLines(mintr_path("mintr_version"))
```

Comment:
You don't need this (or the mintr version) at all, you should use

```suggestion
  as.character(utils::packageVersion("mintr"))
```

here instead